tv_apps/tv_shared/components/smart-banner/script.js

```diff
@@ -41,26 +48,51 @@ window.SmartBanner = (function(win) {
         // We only process 'background-color' because all states have this
         // change.
         if ((evt.propertyName !== 'background-color' &&
+             evt.propertyName !== 'transform' &&
+             evt.propertyName !== 'opacity' &&
              evt.propertyName !== 'width') ||
-            evt.target !== this) {
+             evt.target !== this) {
```

Comment:
No space needed becase we use `evt.target !== this` is aligned to (evt.propertyName......)